src/frontend/api/api.ts

```diff
@@ -28,11 +28,15 @@ export const hentPersonData = (medBarn: boolean): Promise<Person> => {
         .then((response) => response.data);
 };
 
-export const hentArbeidsrettedeAktiviteter = (): Promise<RegisterAktivitet[]> => {
+export const hentArbeidsrettedeAktiviteter = (
+    stønadstype: Stønadstype
+): Promise<RegisterAktivitet[]> => {
+    const url = `${Environment().apiProxyUrl}/aktivitet/v2`;
```

Comment:
Husk å oppdatere denne hvis du endrer backend
```suggestion
    const url = `${Environment().apiProxyUrl}/aktivitet`;
```

Reply:
Ah merging gone to quick 😢